.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Although I am a VSCode user myself (so are several ReadiumJS core contributors too), I think we need to be careful including a config file that could interfere with developers' individual preferences.

Reply:
fair enough! maybe best to gitignore this file then?